docs/logging/key-features.md

```diff
@@ -0,0 +1,96 @@
+---
+id: tofu.logging.key-features
+title: Tofu Logging
+---
+
+
+# Simple logging for your services
+
+## Key features
+
+### Concise syntax
+
+With the implicit instance of `Logging` one can log messages with ease:
+```scala
+import tofu.logging._
+import tofu.syntax.logging._
+val error = new Throwable("Oh no.")
+
+def log[F[_]: Logging: Monad] = for {
+  _ <- info"Hi! I'm logging"
+  _ <- warn"Hello again!"
+  _ <- error"It's me, error!"
+  _ <- errorCause"So sad, I've got an error"(error)
+} yield ()
+```
+
+### Structured and controllable
+You can easily put any necessary values into the structure of the log message with `Loggable`;
+also you can fully control what is logged and what is not:
+
+```scala
+import derevo._
+import tofu.logging._
+
+
+@derive(Loggable)
+case class Payment(id: String, @hidden cardNumber: String, amount: Long)
+
+def log[F[_]: Logging: Monad](payment: Payment) = info"Got payment $payment"
+```
+This will automatically put all the fields of payment into the log structure besides cardNumber field:
+```json
+{
+  "message": "Got payment payment",
+  "level": "INFO",
+  "payment": {
+    "id": "131234234",
+    "amount": 432
+  }
+  
+}
+```
+
+### Context support
+Let's say your effect type has some context — it could be trace id or some domain info:
+```scala
+import cats._
+import cats.effect._
+
+case class TraceId(id: Long)
+
+type TracedIO[A] = ReaderT[TraceId, IO, A]
+```
+
+tofu.logging can extract it and automatically add into the structure of every log message:
```

Comment:
Line states that everything happens "automatically" when in fact you need to define a specific logger and have `loggable` for your context (which is not defined even in this example). I think it would be better to rephrase this as:

"Given that there's a defined loggable instance for this context and provided logger is context aware, every logged message will contain information about context"

Also I think it makes sense to leave a link to `context.md` here.